frontend/src/components/auth/ForgotPasswordEmailModal.tsx

```diff
@@ -0,0 +1,59 @@
+import {
+  Box,
+  Image,
+  Modal,
+  ModalBody,
+  ModalCloseButton,
+  ModalContent,
+  ModalHeader,
+  ModalOverlay,
+  Text,
+} from "@chakra-ui/react";
+import React from "react";
+
+import AuthModalBackground from "../../assets/forgotPasswordModalGraphic.svg";
+
+const ForgotPasswordEmailModal = ({
+  isOpen,
+  onClose,
```

Comment:
nit: create a new interface called `ForgotPasswordEmailModalProps` and put these in there. It'll allow us to document the props properly.